modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
+
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted by specifying a time value in the `retention` field.
+You can also configure the maximum amount of disk space used by the retained metrics data by specifying a size value in the `retentionSize` field.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
+
+Note the following behaviors of these data retention settings:
+
+* The size-based retention policy applies to all data block directories in the `/prometheus` directory, including persistent blocks, write-ahead log (WAL) data, and m-mapped chunks. 
+* Data in the `/wal` and `/head_chunks` directories counts toward the retention size limit, even though data from these directories will never be purged based on size- or time-based retention policies. 
```

Comment:
```suggestion
* Data in the `/wal` and `/head_chunks` directories count toward the retention size limit, even though data from these directories is never purged based on size- or time-based retention policies. 
```

Reply:
```suggestion
Data in the /wal and /head_chunks directories count toward the retention size limit, even though Prometheus never purges data based on size- or time-based retention policies. 
```

Reply:
- Re `data` as singular or plural, the ISG states `Use as a singular noun with a singular verb, even though "data" is the plural of "datum".`
- revised passive voice per your suggestion